src/cursor-tracking-commands/visit.js

```diff
@@ -1,7 +1,13 @@
 let cursorCss
 
 before(() => {
-  cy.readFile('src/css/cursor.css', {
+  let cssPath = 'src/css/cursor-arrow.css'
+
+  if (Cypress.env()['cypress-movie'].cursorTracking.useDot) {
```

Comment:
can we switch from using a boolean to an enum, because we could change things later one.
I am thinking something like
```
let cursorShape = 'dot' | 'arrow' // 'dot' is the default
```

Reply:
Good point, I updated it. Is it fine like this?